apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -2043,16 +2043,21 @@
       "SUCCESS": "Owners successfully removed"
     },
     "DELETE_ENTITY": {
-      "BASIC": "Do you want to delete",
+      "BASIC": "Do you want to {{type}}",
```

Comment:
I would say, in almost all cases "action" is better description of what will be inserted. Maybe except for "CONTROL" where "confirm" may be more suitable.